examples/web-wagmi/src/publications/UseCreateMirror.tsx

```diff
@@ -20,7 +21,7 @@ function MirrorInner({ publisher }: MirrorInnerProps) {
     error: loadingError,
     loading: publicationLoading,
   } = usePublication({
-    publicationId: '0x15-0x028e',
+    publicationId: publicationId('0x15-0x028e'),
     observerId: publisher.id, // important!
```

Comment:
nit not related to this one, `observerId` is not really important anymore, right? We could clean up docs and examples.